### ISSUE TITLE
Scheduled backups to multiple HD/SSD

### ISSUE THREAD

**Issue by Enjymon** [bit-team/backintime#1010]
This is a report on an issue and also a feature request.

BackIntIme doesn't seem to play well with different profiles with scheduled backups to different HD/SSD.

I wanted to set up the following:
- Profile 1: backing up a sub-set of my home daily upon plugin (udev event) to a 1 TB HD
- Profile 2: backing up my full home weekly upon plugin (udev event) to a 4 TB HD

The problem is that only one of the plugin event would trigger a backup (the last added profile would systematically supersede and replace (instead of adding itself to) the previously set profile.

My workaround for the moment is to only have one scheduled backup (Profile 1) and the other (Profile 2) unscheduled and triggered manually.

Also, although each profile has a different destination, they still need to store the backups in two differently numbered directories:
`/DEST1/backintime/<COMPUTER>/<USER>/1`
`/DEST2/backintime/<COMPUTER>/<USER>/2`
which would be unnecessary in that case of different destinations.

It would be nice to have such scheduled profiles to different devices play better with each other.

**Comment by buhtzz** [comment]
@Enjymon There is a new forming maintaining team and we do review all issues. Is this problem still relevant for you, can you reproduce it with a newer or the latest release? Did you find a solution?

Tag: Feature, Support

Seems as a relevant scenario to me that needs further investigation.

**Comment by emtiu** [comment]
I'm not sure I understand the configuration. This is two separate profiles, and they each have have a different externally mounted drive as the destination, and both are set to the schedule „When drive connected“ – correct?

**Comment by Enjymon** [comment]
Hello,

I am just discovering now that a new maintaining team has formed. That's great! Thank you very much all of you for stepping in.

@emtiu Yes you understood correctly.

Unfortunately, my initial post is more than 3 years old now, and as it has not got any reply I have since changed my backup configuration and backup strategy.
If you want to close the issue that's fine with me. Also, because I figured that relying on a drive-connection-event might not be the most convenient for me.
